pymongo/pool.py

```diff
@@ -285,6 +286,20 @@ def _raise_connection_failure(address, error, msg_prefix=None):
     else:
         raise AutoReconnect(msg)
 
+if PY3:
+    def _cond_wait(condition, deadline):
+        timeout = deadline - _time() if deadline else None
+        return condition.wait(timeout)
```

Comment:
Lets add a comment here that `wait()` treats negative timeouts same as `timeout=0`?